scripts/build.sh

```diff
@@ -1,21 +1,20 @@
 #!/bin/bash
 
+PURPLE="\033[0;35m"
+RESET="\033[0m"
```

Comment:
These color variables can be replaced with source utils.sh from minebase. Please tell me what you prefer there.

Reply:
Yeah, I think that would make sense. ^^
Do you want to add this? Otherwise, I'll merge this and fix it myself later.

Reply:
I have added the change.